.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+references:
+    restore_git_cache: &restore_git_cache
+        restore_cache:
+            keys:
+                - v1-git-cache-{{ .Branch }}-{{ .Revision }}
+                - v1-git-cache-{{ .Branch }}-
+                - v1-git-cache-
+    save_git_cache: &save_git_cache
+        save_cache:
+            key: v1-git-cache-{{ .Branch }}-{{ .Revision }}
+            paths:
+                - ".git"
+    restore_docker_compose_cache: &restore_docker_compose_cache
+        restore_cache:
+            key: v1-docker-{{ .Branch }}-{{ checksum "docker-compose.ci.yml" }}
+            paths: ~/caches/images.tar
+    load_docker_image: &load_docker_image
+        run:
+            name: Load docker image
+            command: |
+                set +o pipefail
+                docker load -i ~/caches/images.tar | true
+    run_docker_compose: &run_docker_compose
+        run:
+            name: Run docker compose
+            command: |
+                set -x
+                make env=ci docker-compose-pull
+                make env=ci docker-compose-build
+                make env=ci docker-compose-up-d
+    save_docker_image: &save_docker_image
+        run:
+            name: Save docker image
+            command: |
+                mkdir -p ~/caches
+                docker save $(docker images -q) -o ~/caches/images.tar
+    save_docker_compose_cache: &save_docker_compose_cache
+        save_cache:
+            key: v1-docker-{{ .Branch }}-{{ checksum "docker-compose.ci.yml" }}
+            paths: ~/caches/images.tar
+
+version: 2
+jobs:
+    build:
+        working_directory: ~/workspace
+        machine: true
+        steps:
+            - *restore_git_cache
+            - checkout
+            - *save_git_cache
+            - *restore_docker_compose_cache
+            - *load_docker_image
+            - *run_docker_compose
+            - *save_docker_image
+            - *save_docker_compose_cache
+            - persist_to_workspace:
+                root: .
+                paths:
+                    - .
+    golint:
+        machine: true
+        steps:
+            - attach_workspace:
+                at: .
+            - *restore_docker_compose_cache
+            - *load_docker_image
+            - *run_docker_compose
+            - run:
+                name: Run golint
+                command: make lint
+    vet:
```

Comment:
vet is no longer needed.
https://github.com/bmf-san/Gobel/pull/1/files#diff-1d37e48f9ceff6d8030570cd36286a61R71

diff
https://go-review.googlesource.com/c/go/+/74356

Reply:
done